### PR TITLE
Add tests for models

### PR DESCRIPTION
**Pull request by AleksSenkou**

### CODE REVIEW

File: spec/models/blog/post/meta_information_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+describe Blog::Post::MetaInformation do
+  subject(:post_meta_information) { Blog::Post::MetaInformation.new }
+
+  describe "#post_slug" do
+    it "returns post's slug" do
+      post = double(slug: "slug")
```
Comment (roman-dubrovsky) [inline spec/models/blog/post/meta_information_spec.rb]: @AleksSenkou why didn't you move it to `let` block?

File: spec/models/blog/post_spec.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    describe "#published" do
+      let(:today_published_post) { create :blog_post, :published, published_at: today }
+      let(:not_published_post) { create :blog_post }
+
+      it "retunrs published posts" do
+        expect(Blog::Post.published).to eq [today_published_post]
+      end
+
+      it "returns posts in desc order" do
+        yesterday_published_post = create :blog_post, :published, published_at: yesterday
```
Comment (roman-dubrovsky) [inline spec/models/blog/post_spec.rb]: same

File: spec/models/ckeditor/attachment_file_spec.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+describe Ckeditor::AttachmentFile do
+  subject(:asset) { Ckeditor::AttachmentFile.new }
+
+  describe "#url_thumb" do
+    it "returns image's url thumb" do
+      filename = double
```
Comment (roman-dubrovsky) [inline spec/models/ckeditor/attachment_file_spec.rb]: same

File: spec/models/meta_information_spec.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+describe MetaInformation do
+  context "class methods" do
+    describe "#find_or_new_by_page" do
+      let(:page_name) { "some_page" }
+
+      context "when requested meta information exists" do
+        it "returns meta information" do
+          meta_information = create :meta_information, page: page_name
```
Comment (roman-dubrovsky) [inline spec/models/meta_information_spec.rb]: same

File: spec/models/blog/post/meta_information_spec.rb (hunk 1 above)
Comment (AleksSenkou) [inline spec/models/blog/post/meta_information_spec.rb]: @roman-dubrovsky this var is used only in one `it` block. no need in `let`

Comment (roman-dubrovsky) [inline spec/models/blog/post/meta_information_spec.rb]: @AleksSenkou I think the code is cleaner with `lets` and it's more right by rspec better guides

Comment (roman-dubrovsky) [inline spec/models/blog/post/meta_information_spec.rb]: @AleksSenkou P.s generate slug via faker instead of constant string

Comment (AleksSenkou) [inline spec/models/blog/post/meta_information_spec.rb]: @roman-dubrovsky yeah, `let` is really better here.

Comment (AleksSenkou) [inline spec/models/blog/post/meta_information_spec.rb]: @roman-dubrovsky don't sure about Faker here. Why not `double`? Why not static string?

File: spec/models/blog/post/meta_information_spec.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+describe Blog::Post::MetaInformation do
+  subject(:post_meta_information) { Blog::Post::MetaInformation.new }
+
+  describe "#post_slug" do
+    let(:post) { double(slug: "slug") }
```
Comment (roman-dubrovsky) [inline spec/models/blog/post/meta_information_spec.rb]:
```
AleksSenkou an hour ago Member
@roman-dubrovsky don't sure about Faker here. Why not double? Why not static string?
```

I suggest to move "slug" to separate let and generate it randomly. Like

```
let(:slug) { Faker::Lorem.word }
let(:post) {  double(slug: slug) }

...

expect(post_meta_information.post_slug).to eq slug
```

What do you think?

Comment (AleksSenkou) [inline spec/models/blog/post/meta_information_spec.rb]: @roman-dubrovsky why not just `double`
```
let(:slug) { double }
```

Comment (roman-dubrovsky) [inline spec/models/blog/post/meta_information_spec.rb]: @AleksSenkou it's ok if it'll be work

**Comment by AleksSenkou** [comment]
@roman-dubrovsky approve this PR please

File: spec/models/blog/post_spec.rb [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+        expect(Blog::Post.published).to eq [today_published_post, yesterday_published_post]
+      end
+    end
+  end
+
+  describe "#to_param" do
+    it "returns slug" do
+      allow(post).to receive(:slug).and_return("slug")
```
Comment (roman-dubrovsky) [inline spec/models/blog/post_spec.rb]: 😱 hardcoded string?

File: spec/models/meta_information_spec.rb [hunk 7]
```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+describe MetaInformation do
+  context "class methods" do
+    describe "#find_or_new_by_page" do
+      let(:page_name) { "some_page" }
```
Comment (roman-dubrovsky) [inline spec/models/meta_information_spec.rb]: @AleksSenkou can you change it from hardcoded string?

File: spec/models/blog/post_spec.rb (hunk 2 above)
Comment (roman-dubrovsky) [inline spec/models/blog/post_spec.rb]: @AleksSenkou also if I were you I would remove the first test (`retunrs published posts`) or move this variable to let and write like

```
describe `for some posts` do
  let!(: yesterday_published_post) {  create :blog_post, :published, published_at: yesterday }
  it "returns posts in desc order" do
    expect(Blog::Post.published).to eq [today_published_post, yesterday_published_post]
  end
end
```

I prefer the way with removing the first test. what do you think?

**Comment by roman-dubrovsky** [comment]
@AleksSenkou I've wirtten comment above ☝️

File: spec/models/blog/post_spec.rb (hunk 2 above)
Comment (AleksSenkou) [inline spec/models/blog/post_spec.rb]: @roman-dubrovsky it's not way of one dependence
Here first test checks that method returns published posts, second - orders posts

It's important in order to split logic and find bugs faster in the future

**Comment by AleksSenkou** [comment]
@roman-dubrovsky I've fixed hardcoded strings. Check it ;)

File: spec/models/blog/post_spec.rb (hunk 2 above)
Comment (roman-dubrovsky) [inline spec/models/blog/post_spec.rb]: @AleksSenkou xz, I;m not sure that the first test is unnecessary (and I know that It is wrote in Clean Code but for me it's not best example for this rule)

**Pull request merged**